docs/src/main/docs/tracing/01_tracing.adoc

```diff
@@ -57,3 +57,72 @@ ServerConfiguration.builder()
 ----
 <1> The name of the application (service) to associate with the tracing events
 <2> The endpoint for tracing events, specific to the tracer used, usually loaded from Config
+
+=== Configuring Tracing with Helidon MP
+There is an additional dependency that adds Tracing support to MP server and Jersey client:
+
+[source,xml,subs="verbatim,attributes"]
+.MP Tracing
+----
+<dependency>
+    <groupId>io.helidon.microprofile.tracing</groupId>
+    <artifactId>helidon-microprofile-tracing</artifactId>
+</dependency>
+----
+
+The minimal required configuration is `tracing.service` that contains
```

Comment:
As I understand this part explains how to configure tracing for Helidon MP. I'd rather suggest a specific section about configuration.

Show all the configuration options with a description, and specify which ones are optional or required ; A table is a good fit unless there is a large number of configuration options, in this case I'd show the most common one and refer to the full list in the javadocs with a link.

Reply:
Added section with configuration options.